cmake/modules/unittest.cmake

```diff
@@ -58,8 +58,22 @@ include(${ZEPHYR_BASE}/cmake/kobj.cmake)
 add_dependencies(test_interface ${KOBJ_TYPES_H_TARGET})
 gen_kobj(KOBJ_GEN_DIR)
 
+# Generates empty header files to build
+file(TOUCH ${APPLICATION_BINARY_DIR}/zephyr/include/generated/devicetree_generated.h)
```

Comment:
is it more elegant to do a list + foreach instead?

Reply:
yes, I guess.

Reply:
converted to list + foreach